rust/src/rest.rs

```diff
@@ -1,3 +1,5 @@
+//! A RESTful API for the PET protocol interactions.
```

Comment:
This is a detail, but I'm not sure our API qualifies as RESTful. We don't have Create, Update, or Delete operations, we use a custom binary format for the messages, some GET requests carry a binary payload, etc. I think we should just talk about "HTTP API".

Reply:
true; all in all, it's pretty unRESTful :) I'll open a PR to fix.